sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -20,84 +20,45 @@ autorest
 ### Settings
 
 ```yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/analyzeconversations.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/release-cognitiveservices-Language-2022-05-01-preview/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json
 output-folder: ../azure/ai/language/conversations
 namespace: azure.ai.language.conversations
 package-name: azure-ai-language-conversations
 license-header: MICROSOFT_MIT_NO_VERSION
 clear-output-folder: true
 no-namespace-folders: true
 python: true
+tag: release_2022_05_01_preview
+openapi-type: data-plane
 title: ConversationAnalysisClient
 version-tolerant: true
 models-mode: msrest
-package-version: 1.0.0b3
+package-version: 1.0.0b4
 add-credential: true
 credential-default-policy-type: AzureKeyCredentialPolicy
 credential-key-header-name: Ocp-Apim-Subscription-Key
 black: true
+modelerfour:
+  lenient-model-deduplication: true
 ```
 
-### Remove intermediary object from analyze operation call
 
-```yaml
-directive:
-    - from: swagger-document
-      where: $["paths"]["/:analyze-conversations"]["post"]
-      transform: >
-          $["operationId"] = "analyzeConversation";
-```
-
-### Rename body to tasks
+### Rename 'duplicate schema' errors in 'Task State'
 
 ```yaml
 directive:
     - from: swagger-document
-      where: $["paths"]["/:analyze-conversations"]["post"]
+      where: $["definitions"]["TaskState"]["properties"]
       transform: >
-        $["parameters"][1]["x-ms-client-name"] = "task";
+        $["status"]["x-ms-enum"]["name"] = "TaskStateEnum";
 ```
 
-### Rename 'confidenceScore' in Qna intent result
+### Rename 'duplicate schema' errors in 'Job State'
 
 ```yaml
 directive:
     - from: swagger-document
-      where: $["definitions"]["KnowledgeBaseAnswer"]["properties"]["confidenceScore"]
+      where: $["definitions"]["JobState"]["properties"]
```

Comment:
These really should be in the swagger. Lots of toosl are affected. There is a push to set `x-ms-client-name` to disambiguate. /cc @abhahn